doc/modules/cross_validation.rst

```diff
@@ -116,16 +116,15 @@ a model and computing the score 5 consecutive times (with different splits each
 time)::
 
   >>> from sklearn.model_selection import cross_val_score
-  >>> clf = svm.SVC(kernel='linear', C=1)
+  >>> clf = svm.SVC(kernel='linear', C=1, random_state=42)
   >>> scores = cross_val_score(clf, X, y, cv=5)
   >>> scores
-  array([0.96..., 1.  ..., 0.96..., 0.96..., 1.        ])
+  array([0.96666667, 1.        , 0.96666667, 0.96666667, 1.        ])
```

Comment:
Let's leave the `...` in here:

```suggestion
  array([0.96..., 1. , 0.96..., 0.96..., 1. ])
```

Reply:
Oh whoops I just thought the exact output would prevent any possible confusion, I'll revert it.